ONSdigital/eq-author-prototypes
# Replace all instances of term Response with Answer

## Description
Fixes #19.
This PR replaces all references to the term "Response" with "Answer".

## How to test
Launch the authoring prototype,
Observe that all references to the term response have been replaced.
Ensure that the application still works as expected.
